packages/config/tests/load/snapshots/tests.js.md

```diff
@@ -381,15 +381,15 @@ Generated by [AVA](https://ava.li).
 
     `{␊
       "branch": "branch",␊
-      "buildDir": "/file/path",␊
+      "buildDir": "",␊
```

Comment:
This is expected because we're running the fixture from process.cwd(), so the relative path will be the empty string.